sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/UnsafeRow.java

```diff
@@ -283,6 +284,7 @@ public void setDecimal(int ordinal, Decimal value, int precision) {
         // keep the offset for future update
         Platform.putLong(baseObject, getFieldOffset(ordinal), cursor << 32);
       } else {
+
```

Comment:
nit: remove empty line

Reply:
ditto: Please ignore this if it's invalid. I reviewed against the last commit and not sure why Github says "Outdated".